src/js/share/share.js

```diff
@@ -612,4 +614,38 @@ export class Share {
 
     return col;
   }
+
+  /**
+   * handle click events on scene headers and card headers
+   * change chevron direction and set title bold/normal
+   * @param {Element} sceneElement
+   */
+  changeHeaderDesign(sceneElement) {
+    const parentContainer = sceneElement.parentElement.parentElement;
+    const ariaExpanded = sceneElement.getAttribute('aria-expanded');
+    const divElement = sceneElement.children[0];
+    const iElement = sceneElement.children[1];
+    //set design for current element
+    if (iElement.textContent.includes('chevron_left') && ariaExpanded === 'false') {
+      iElement.textContent = iElement.textContent.replace("chevron_left", "expand_more");
+      divElement.style.fontWeight = "bold";
+    }
+    else {
+      if (iElement.textContent.includes('expand_more') && ariaExpanded === 'true') {
+        iElement.textContent = iElement.textContent.replace("expand_more", "chevron_left");
+        divElement.style.fontWeight = "normal";
+      }
+      else {
+        console.error("can't change chevron and header title: " + sceneElement.getAttribute('class'));
+      }
+    }
+    //set design for all other elements in the same parent div
+    for(let i = 0; i < parentContainer.children.length; i++) {
+      const element = parentContainer.children[i].children[0];
+      if (element !== sceneElement) {
+        element.children[0].style.fontWeight = "normal";
+        element.children[1].textContent = iElement.textContent.replace("expand_more", "chevron_left");
+      }
+    }
+  }
```

Comment:
this function is callthis function is called by the click event handler. 
Two steps:
1. change title and chevron of current element
2. change other elements in the parent container (when one script/scene is open and the user selects another one, then the previous one will be closed)ed by the click event handler. 
Two steps:
1. change title and chevron of current element
2. change other elements in the parent container (when one script/scene is open and the user selects another one, then the previous one will be closed)